lib/request/index.js

```diff
@@ -15,21 +30,76 @@ module.exports = function (options, callback) {
         return callback(new Error('expecting a valid curl command to run'));
     }
 
-    const runner = new runtime.Runner();
+    const runner = new runtime.Runner(),
+        emitter = new EventEmitter();
+
+    // store summary object and other relevant information inside the emitter
+    emitter.summary = new RunSummary(emitter, options);
 
     util.convertCurltoCollection(options.curl, function (err, curlCollection) {
         if (err) {
-            callback(err); // eslint-disable-line
+            return callback(err);
         }
 
-        runner.run(curlCollection, {}, function (err, runResult) {
-            console.log(err); // eslint-disable-line
-            console.log(runResult); // eslint-disable-line
+        runner.run(curlCollection, {
+            requester: {
+                verbose: true
```

Comment:
Please use verbose only when the user gives the flag, otherwise, we are just processing for more than what we need.